src/main/java/uk/gov/hmcts/juror/api/juror/domain/WelshCourtLocation.java

```diff
@@ -38,6 +38,12 @@ public class WelshCourtLocation implements Serializable, ICourtLocation {
     @Column(name = "loc_name")
     private String locCourtName;
 
+    /**
+     * Court name.
```

Comment:
might be worth explaining why this is also court name as we have the same above?

Reply:
Updated :)